doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -96,14 +100,19 @@ is -1.
 When called on a client B<ssl>, SSL_get_shared_group() has no meaning and
 returns -1.
 
+SSL_get_negotiated_group() returns the NID of the negotiated group on a
+TLSv1.3 connection for Key exchange. Or it returns NID_undef if no negotiated
+group.
+
 =head1 SEE ALSO
 
 L<SSL_CTX_add_extra_chain_cert(3)>
 
 =head1 HISTORY
 
 The curve functions were added in OpenSSL 1.0.2. The equivalent group
-functions were added in OpenSSL 1.1.1.
+functions were added in OpenSSL 1.1.1. SSL_get_negotiated_group() function were
```

Comment:
"The SSL_get_negotiated_group() function was added..."

Reply:
Fixed